components/concept/engine/src/main/java/mozilla/components/concept/engine/prompt/PromptRequest.kt

```diff
@@ -346,6 +347,21 @@ sealed class PromptRequest(
         override val onDismiss: () -> Unit
     ) : PromptRequest(), Dismissible
 
+    /**
+     * Value type that represents a request for a select address prompt.
+     *
+     * This prompt is triggered by the user focusing on an address field.
+     *
+     * @property addresses List of addresses for the user to choose from.
+     * @property onConfirm Callback used to confirm the selected address.
+     * @property onDismiss Callback used to dismiss the address prompt.
+     */
+    data class SelectAddress(
```

Comment:
This is just a preference thing, can you move this closer to the other SelectXX prompts